ts/src/binance.ts

```diff
@@ -8431,6 +8681,29 @@ export default class binance extends Exchange {
         return { 'url': url, 'method': method, 'body': body, 'headers': headers };
     }
 
+    getExceptionMarketTypeByUrl (url, exactOrBroad) {
```

Comment:
`getExceptionMarketTypeByUrl` seems like we're getting only the "exception's market type" and not the exceptions itself, maybe `getExceptionsByUrl` is enough?

Reply:
agggrhh, shame on me, how i missed that name... pushing next update soon